src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+  
+    &:hover {
+      color: #00acdc;
+      transition: $time;
+    }
```

Comment:
https://user-images.githubusercontent.com/94289038/201955917-13185e3e-5912-4ab2-a3ea-05e374934c91.mov

now your transition works only on hover move transition on element and define properties as well
fix it in both places